sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2702,7 +2702,7 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
 
       val e = intercept[AnalysisException](sql("SELECT v.i from (SELECT i FROM v)"))
       assert(e.message ==
-        "cannot resolve '`v.i`' given input columns: [__auto_generated_subquery_name.i]")
```

Comment:
I think the problem here is the out-most backticks, do you know where we add it?

Reply:
The out-most backticks is added in 
```case _```, in ```case class UnresolvedAttribute(nameParts: Seq[String])```
```
  override def sql: String = name match {
    case ParserUtils.escapedIdentifier(_) | ParserUtils.qualifiedEscapedIdentifier(_, _) => name
    case _ => quoteIdentifier(name)
  }
```
the ```nameParts``` is a Seq of ```"v" "i"```, name is ```v.i```